deploy/template.yaml

```diff
@@ -101,7 +101,7 @@ Mappings:
   EnvironmentVariables: # This is all the environment specific environment variables that don't belong in globals.
     dev:
       YOTIBASEURL: "https://f2f-yoti-stub-yotistub.review-o.dev.account.gov.uk"
-      YOTISDK: "MockSDK"
+      YOTISDK: "1f9edc97-c60c-40d7-becb-c1c6a2ec4963"
```

Comment:
Conscious that our repos are public, is this something that we're definitely happy to have in the public domain?